PhysicsTools/NanoAOD/python/photons_cff.py

```diff
@@ -101,7 +133,7 @@
         pixelSeed = Var("hasPixelSeed()",bool,doc="has pixel seed"),
```

Comment:
If IDs are not nested, also the definition of cutBased as `userInt('cutbasedID_loose')+userInt('cutbasedID_medium')+userInt('cutbasedID_tight')` may have to be changed

(github doesn't allow me to put the comment on that line, since it's not modified in this PR)

Reply:
Modified to a bitmap, changing the variable name for more safety in its usage